waspc/data/Generator/templates/sdk/wasp/client/test/vitest/helpers.tsx

```diff
@@ -7,7 +7,7 @@ import { render, RenderResult, cleanup } from '@testing-library/react'
 import { QueryClient, QueryClientProvider } from '@tanstack/react-query'
 import { beforeAll, afterEach, afterAll } from 'vitest'
 import { Query } from 'wasp/client/operations/core'
-import config from 'wasp/core/config'
+import { config } from 'wasp/client'
```

Comment:
Maybe add this rewrote to the code mod as well? Probably some users used it.

Reply:
Nice thinking, but it was never a part of the public API (and we had the same path for the server config, which was).

Therefore, the code mod is up to date.

Reply:
What's the name of the principle you like to mention? The public API is what users think there public API is 😂

Reply:
Hyrum's law, but I don't think we understood each other.

In 11.8, the import `wasp/config` resolved to two symbols (depending on which folder it was in). One of those symbols was part of the public API (the server config), the other wasn't (the client config).

Codemod does not support mapping a single import (`wasp/config`) to two different new imports (`wasp/client` and `wasp/server`) depending on where it finds the file that uses them.
It currently maps all config imports from `wasp/config` imports to `wasp/server` imports (which was a documented feature) regardless of where it finds them.

I could therefore either:
1. Change codemod to map `wasp/config` imports to `wasp/client` imports, which would be correct for the undocumented client config imports, but incorrect for the documented server config imports (which, of course, doesn't make much sense :smile:).
2. Update codemod to be context-aware and map `wasp/config` to the correct path depending on whether it's inside the `src/client` or `src/server` directory. This is obviously the best solution, but would take a lot of work for a feature we technically never had, so I haven't considered it.

Reply:
Oh okay, I was thinking along the lines of supporting users to migrate from `'wasp/core/config'` to `wasp/client`. But I guess that would be an exception for our codemod, no? It upgraded users from 0.11 to 0.12 not 0.12 to 0.13!

I agree with what you wrote, no need to go on a complicated refactor to support this one import.